<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WareSrcWidget</class>
 <widget class="QWidget" name="WareSrcWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="Toolbar_Layout"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="WareSrcFileCollection">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Messages:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="openfluid::ui::waresdev::WareSrcMsgViewer" name="WareSrcMessageWidget">
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>180</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>openfluid::ui::waresdev::WareSrcMsgViewer</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">openfluid/ui/waresdev/WareSrcMsgViewer.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
